Enterprise-CMCS/macpro-onemac
# PI01.02: 25703: MASTER-Fix Email Content for Medicaid SPA Submission Notifications

Story: https://qmacbis.atlassian.net/browse/OY2-25703
Endpoint: [Click here](https://d2q622g4esaec3.cloudfront.net)

### Details
Email static text hotfix got reverted.  Add the work again.

### Changes
- Duplicated the changes to emails found in develop branch 

### Implementation Notes
- PR for work in develop: https://github.com/Enterprise-CMCS/macpro-onemac/pull/1339
- original story: 24938

### Test Plan - taken from earlier PR
1. Login as a state submitter for which you have access to the emails
2. Navigate to the dashboard
3. Use the "New Submission->State Plan Amendment (SPA)->CHIP SPA->All Other CHIP SPA Submissions"
4. Submit the form
5. Check your email and verify that the closing remarks of the message match the Confluence page (other things are different because the Confluence page hasn't been updated in a while)
